unified_planning/test/__init__.py

```diff
@@ -116,5 +117,27 @@ def wrapper(*args, **kwargs):
         return wrapper
 
 
+class skipIfModuleNotInstalled(object):
+    """Skip a test if there are no oneshot planner for the given problem kind."""
+
+    def __init__(self, module_name: str):
+        self.module_name = module_name
+
+    def __call__(self, test_fun):
+        msg = f"no module named {self.module_name} installed"
+        cond = False
+        try:
+            importlib.import_module(self.module_name)
```

Comment:
A better option would be to use `find_spec` that does not import the module, but generally OK. 
See: https://docs.python.org/3/library/importlib.html#importlib.util.find_spec